added weekly-2023-05-18.adoc (#122)
jbossorg/jbossorg.github.io: src/content/posts/weekly-2023-05-18.adoc

== How to use the new OpenShift quick starts to deploy JBoss EAP

link:https://developers.redhat.com/articles/2023/05/15/how-use-new-openshift-quick-starts-deploy-jboss-eap[How to use the new OpenShift quick starts to deploy JBoss EAP], by Philip Hayes

This article showcases the latest JBoss EAP quick start, specifically created to assist developers who are already familiar with conventional JBoss EAP deployments. Its purpose is to provide a comprehensive walkthrough on constructing and deploying application images on OpenShift. The quick start offers valuable guidance on utilizing Helm to generate the necessary build configs, deployment configs, and external routes for building and deploying JBoss EAP applications on OpenShift.

== Integrate Excel with Drools on OpenShift with Knative and Quarkus

link:https://blog.kie.org/2023/05/integrate-excel-with-drools-on-openshift-with-knative-and-quarkus.html[Integrate Excel with Drools on OpenShift with Knative and Quarkus], by Matteo Mortari

> In this blog post, Matteo shares the results of a technical exploration of bringing together and connecting different technologies and platforms. At the end of the day, he combined things like regular spreadsheet applications (like Excel), serverless platforms (Knative on OpenShift), and our rule engine Drools to see how they could work together.

Reviewer: a little redundant to say "bringing together and connecting" imo

> == Podman Desktop Beginners' Guide

Reviewer: ```suggestion
== Podman Desktop Beginner's Guide
```

link:https://www.mastertheboss.com/soa-cloud/docker/podman-desktop-a-beginners-guide-to-containerization/[Podman Desktop Guide], by Francesco Marchioni

In this tutorial, I'm introducing the Podman desktop UI which simplifies the usage and management of Container images. As an example, we will learn how to pull, start and monitor WildFly Container image with just a few clicks!

== What’s new in Red Hat’s migration toolkit for applications 6.1

link:https://developers.redhat.com/articles/2023/05/10/whats-new-red-hats-migration-toolkit-applications-61[What’s new in Red Hat’s migration toolkit for applications 6.1], by Yashwanth Maheshwaram

> Red Hat Migration Toolkit is an essential tool to simplify the upgrade or migration of a large set of Enterprise applications. Stay updated with the latest news (video included) in this article.

Reviewer: ```suggestion
Red Hat Migration Toolkit is an essential tool to simplify the upgrade or migration of a large set of enterprise applications. This article gives you an update with the latest news and includes a great demo video.
```
